mmv1/products/iam3/product.yaml

```
@@ -0,0 +1,20 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: 'IAM3'
+display_name: 'Cloud IAM'
+versions:
+  - name: 'ga'
+    base_url: 'https://iam.googleapis.com/v3/'
```

I can only see the [a beta 'v3beta' version of iam3](https://cloud.google.com/iam/docs/reference/rest/v3beta/organizations.locations.principalAccessBoundaryPolicies) in the public docs, so I think we need to ensure this new resource uses those endpoints and is only added to the google-beta provider. 

```suggestion
  - name: 'beta'
    base_url: 'https://iam.googleapis.com/v3beta/'
```